tests/ui_tools/test_messages.py

```diff
@@ -1725,6 +1737,7 @@ def test_reactions_view(
         expected_attributes,
     ):
         self.model.user_id = 1
+
```

Comment:
Unrelated addition.

Reply:
Probably a mistyping. I'll delete it with a new commit before I squash all the commits together?

Reply:
Yes, or you can use eg `git commit --amend -a` to update the previous commit code and text, whether normally at the tip of a branch, or if in the middle of an interactive rebase.